index.html

```diff
@@ -173,30 +173,107 @@ <h2>
-        error in changing state. They are named <dfn>pointerlockchange</dfn>
-        and <dfn>pointerlockerror</dfn>. If pointer lock is entered or exited
-        for any reason a <a>pointerlockchange</a> event must be sent.
-      </p>
-      <p>
-        <a>User agents</a> must deliver these events by <a data-lt=
-        "queue a task">queuing a task</a> to <a>fire an event</a> of the
-        appropriate name with its <code>bubbles</code> attribute [[DOM]] set to
-        false to the pointer lock <a>target</a> element's [=Node/node
-        document=].
-      </p>
-      <p class="note">
-        Magnification software increases the size of content on the screen. It
-        uses the mouse to move the magnified point of focus around. When a
-        pointer lock is initiated, the magnification software needs to switch
-        to using the keyboard to move the magnified point of focus around
-        instead. When a pointerlockchange event is fired, web browsers
-        therefore need to make sure the event is communicated to assistive
-        technologies like screen magnifiers.
-      </p>
+      <section>
+        <h3>
+          The <dfn>Pointer Lock State</dfn> Definition
+        </h3>
+          <p><a>Pointer Lock State</a> is the state where a single DOM element, which
+            we will call the <dfn>target</dfn>, receives all mouse events and the cursor
+            is hidden.</p>
+          <p>
+            Once in the <a>Pointer Lock State</a> the <a>user agent</a> must fire all
+            relevant user generated {{MouseEvent}} events (for example:
+            `mousemove`, `mousedown`, `mouseup`, `click`, and `wheel`)
+            [[ui-events]] to the <a>target</a> of pointer lock, and not fire
+            mouse events to other elements. Events that require the concept of
+            a mouse cursor must not be dispatched (for example: `mouseover`,
+            `mouseout`, `drag`, and `drop`).
+          </p>
+          <p>
+            <dl data-dfn-for="PointerLockOptions" data-link-for=
+            "PointerLockOptions">
+              While in the <a>Pointer Lock State</a> if <a>unadjustedMovement</a> is
+              <code>true</code> the event coordinates must not be affected by the
+              underlying platform behaviors such as mouse acceleration. In other
+              words, UA must use the APIs provided by the underlying platform to
+              guaranteee getting the raw events. If <a>unadjustedMovement</a>
+              is <code>undefined</code> or <code>false</code> UA must rely on the
+              default behavior of the underlying platform regarding the mouse
+              acceleration.
+            </dl>
+          </p>
+          <p>
+            In the <a>Pointer Lock State</a> the system mouse cursor must be hidden.
+            Movement and button presses of the mouse must not cause the window
+            to lose focus.
+          </p>
+          <p>
```

Comment:
```suggestion
          <p class="note">
```

Since this is kind of a side-note that is about something related, but not part of the normative definition of the Pointer Lock State itself.

Reply:
done